modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/decisions_controller.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+    module V0
+        class DecisionsController < ApplicationController
+            before_action :set_power_of_attorney_request, only: [:show, :create]
+
+            def create
+                if @power_of_attorney_request.resolution.present?
+                    render json: { error: 'Resolution already exists' }, status: :unprocessable_entity
+                    return
+                end
+        
+                creator_id = @current_user.id
+        
+                type = power_of_attorney_request_params[:declination_reason].present? ? PowerOfAttorneyRequestDecision::Types::DECLINATION :  PowerOfAttorneyRequestDecision::Types::ACCEPTANCE
+                decision = PowerOfAttorneyRequestDecision.new(type:, creator_id:)
+        
+                ActiveRecord::Base.transaction do
+                    if decision.save
+                        resolution = PowerOfAttorneyRequestResolution.new(
+                            power_of_attorney_request: @power_of_attorney_request,
+                            resolving: decision,
+                            reason: power_of_attorney_request_params[:declination_reason]
+                        )
+        
+                        if resolution.save
+                            render json: decision, status: :ok
+                        else
+                            raise ActiveRecord::Rollback
```

Comment:
I don't know that there are any normal expressions where one would need to do something this low-level from the framework.